test/powershell/Modules/Microsoft.PowerShell.Utility/Invoke-Item.Tests.ps1

```diff
@@ -37,14 +37,58 @@ Describe "Invoke-Item" -Tags "CI" {
         $testfile = Join-Path $TestDrive testfile.txt
     }
 
-    It "Should invoke a text file without error" -Skip:($IsWindows -and $IsCoreCLR) {
+    It "Should invoke a text file without error on non-Windows" -Skip:($IsWindows) {
         $debugfn = NewProcessStartInfo "-noprofile ""``Invoke-Item $testfile`n" -RedirectStdIn
         $process = RunPowerShell $debugfn
         EnsureChildHasExited $process
         $process.ExitCode | Should Be 0
     }
 
+<#
     It "Should throw not supported on Windows with .NET Core" -Skip:($IsLinux -or $IsOSX -or !$IsCoreCLR) {
         { Invoke-Item $testfile }| Should Throw "Operation is not supported on this platform."
     }
+#>
+}
+
+Describe "Invoke-Item tests on Windows" -Tags "CI","RequireAdminOnWindows" {
+    BeforeAll {
```

Comment:
The whole content of BeforeAll block should be wrapped in the `if` statement for skipping the non-windows platforms. You can use this technique (or something similar) for skipping: https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#skipping-tests-in-bulk 
It's better to avoid copying the skipping condition everywhere and use a variable instead